scripts/_generate/index.js

```diff
@@ -71,18 +71,18 @@ async function generateScss () {
     return rendered.css.toString();
 }
 
-async function generateMarkup () {
+function generateMarkupSync () {
     console.log('Generating HTML');
-
     let cfg = {
         files: [
             '*.html',
             '**/*.html',
+            '!files/**', // ignore docs/files
```

Comment:
Without this, we'd lose some files in `public/files/*` due to race conditions with async behaviors of the `copy` and `generate` logic.